tests/ProjectTest.php

```diff
@@ -142,9 +142,16 @@ public function testGetPath()
         $actual = $this->project->getPath();
         $this->assertSame($expect, $actual);
 
-        $expect = $this->path . '/foo/bar/baz.txt';
-        $actual = $this->project->getPath('foo/bar/baz.txt');
-        $this->assertSame($expect, $actual);
+        if (DIRECTORY_SEPARATOR != '\\') {
```

Comment:
Alternatively we could  use DIRECTORY_SEPARATOR and make the string path. So no changes in test code.

Reply:
Refactored.